src/test/java/net/openhft/chronicle/threads/LongPauserBenchmark.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 Higher Frequency Trading
+ *
+ * http://chronicle.software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.openhft.chronicle.threads;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.util.concurrent.TimeUnit;
+
+
+public final class LongPauserBenchmark {
```

Comment:
This used to be a @Test but is really a benchmark so I moved it to a main method instead.